authserver/db/models/models.py

```diff
@@ -134,7 +133,7 @@ class Meta:
     lastname = fields.String(required=True)
     organization = fields.String(required=True)
     email_address = fields.Email(required=True)
-    telephone = fields.String()
+    telephone = fields.String(required=True)
```

Comment:
I think there's some logic in the POST endpoint for a user that looks for the existence of a telephone number. This may have come out of some conversation somewhere in the past about having some way of contacting a user. Basically, unit test failed even before requiring the telephone number if the telephone number wasn't present. I'm not a fan of requiring this.

Reply:
Oh, it looks like post method has this: 
https://github.com/brighthive/authserver/blob/c512659ff39a56a61c6359ba3296f9db9c9cb4d5/authserver/api/user.py#L120-L123

Would it be worth adjusting the above ^^^ since telephone is a required attribute?

(I'll have to update Facet user forms and logic – so, that's why I'm paying attention!)